datadragon/data_dragon.go

```diff
@@ -318,7 +318,7 @@ func (c *Client) GetSummonerSpell(id string) (SummonerSpell, error) {
 		return SummonerSpell{}, err
 	}
 	for _, summonerSpell := range summonerSpells {
-		if summonerSpell.ID == id {
+		if summonerSpell.Key == id {
```

Comment:
same here

Reply:
Same rationale i mentioned for the other spot: It's supplied as an integer, which maps to key (which is a quoted integer). ID is a string name which would never evaluate true. Of note now that i think:
With this being the case, what if we changed these two method to use int parameters and to change the structs to unmarshal to an int?